vision/cloud-client/web/web_entities.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python
+
+# Copyright 2017 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Demonstrates web detection using the Google Cloud Vision API.
+
+This sample prints only web_entities allowing the user to specify
+whether to include geo_results, which are based on GPS information
+included in the image.  Use the "--include-geo-results" flag to turn on
+the geo entities feature.
+
+Example usage:
+  python web_entities.py https://goo.gl/X4qcB6 --include-geo-results
+  python web_entities.py ../detect/resources/city.jpg --include-geo-results
+  python web_entities.py gs://your-bucket/image.png --include-geo-results
+"""
+# [START imports]
+import argparse
+import io
+
+from google.cloud import vision
+from google.cloud.vision import types
+# [END imports]
+
+
+# [START def_annotate_web_entities]
+def annotate_web_entities(path, include_geo_results=False):
+    """Returns web annotations given the path to an image."""
+    client = vision.ImageAnnotatorClient()
+
+    if path.startswith('http') or path.startswith('gs:'):
```

Comment:
I really don't like branches like this in samples, I prefer separate samples for local images and remote images.

Reply:
Good idea - I copied the pattern from `web_detect.py` without thinking much into it.  I will update both files when I get the chance to.